frontend/src/features/mapSideBar/hooks/useQueryParamSideBar.tsx

```diff
@@ -12,6 +12,7 @@ export enum SidebarContextType {
   ADD_BUILDING = 'addBuilding',
   ADD_BARE_LAND = 'addBareLand',
   ADD_ASSOCIATED_LAND = 'addAssociatedLand',
+  ADD_SUBDIVISION_LAND = 'addSubdivisionLand',
```

Comment:
so there will be no "view" or "update" for subdivisions?

Reply:
view and update reuse the existing land view and update logic.